docs/install/full-install.rst

```diff
@@ -0,0 +1,14 @@
+=================
+Full Installation
+=================
+
+For a full Flocker installation, including the Flocker plugin for Docker, the following manual instructions take you through everything you need to do:
+
+.. XXX this introduction could be improved with an image. See FLOC-2077
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
:question: Another time I'd like a style guide - how do we choose what the `:maxdepth:` should be for each toctree?